lib/ui/recovery_key_dialog.dart

```diff
@@ -128,6 +139,17 @@ class _RecoveryKeyDialogState extends State<RecoveryKeyDialog> {
               Text(
                 "please save this in a safe place",
               ),
+              Padding(padding: EdgeInsets.all(8)),
+              CheckboxListTile(
+                  contentPadding: EdgeInsets.zero,
+                  controlAffinity: ListTileControlAffinity.trailing,
+                  title: Text('show human readable key'),
+                  value: _showMnenomicCode,
+                  onChanged: (value) {
+                    setState(() {
+                      _showMnenomicCode = value;
+                    });
+                  }),
```

Comment:
Need to take a call whether we want to show the option to view non-mnemonic recovery key or not.
Accordingly we will either add similar option on web or remove it from the mobile.

Reply:
Let's always show the mnemonic code, since the effort to write it down is lesser and that to save is the same.